add_user.php

```diff
@@ -78,11 +78,6 @@
 									</center>
 								</div>
                                 
-                <!-- This class is for my footer-->	
```

Comment:
I removed all footers on all admin pages to be identical layout to the login and register pages